lighthouse-core/test/report/html/renderer/util-test.js

```diff
@@ -9,6 +9,10 @@ const assert = require('assert');
 const Util = require('../../../../report/html/renderer/util.js');
 const sampleResult = require('../../../results/sample_v2.json');
 
+// Require i18n to make sure Intl is polyfilled in Node without full-icu.
```

Comment:
maybe worth noting that this code won't actually be run in node though for a real report but it's OK because browser should support it?